pages/api/services/home-page-service.js

```diff
@@ -92,10 +84,10 @@ export const getHomepageData = async user => {
 				balances[name] = lastCredit[0]['from_balance'];
 			}
 			// if it's had any debits
-		} else if (lastDebit.length > 0) {
+		} else if (lastDebit.length) {
```

Comment:
This 'debit' and 'credit' language isn't got yet? Do a search across the repo to double check for all